doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-groups-ref.adoc

```diff
@@ -0,0 +1,90 @@
+[id='security-management-rest-api-groups-ref_{context}']
+= Groups
+
+The Security Management REST API supports the following endpoints for managing groups in {CENTRAL}. The Security Management REST API base URL is `\http://SERVER:PORT/{URL_COMPONENT_CENTRAL}/rest/`. All requests require HTTP Basic authentication or token-based authentication for the `admin` user role.
+
+[GET] /groups::
++
+--
+Returns all groups in {CENTRAL}.
+
+.Example server response (JSON)
+[source,json]
+----
+[
+  	{
+		"group1"
+	},
+	{
+		"group2"
+	}
+]
+----
+--
+[POST] /groups::
++
+--
+Creates a group in {CENTRAL}.
```

Comment:
Although in the example, we are creating a group with some user, I would mention here that when creating a group, at least one user has to be assigned to the group